plugins/jdbc/src/main/java/com/megaease/easeagent/plugin/jdbc/interceptor/tracing/JdbcStmTracingInterceptor.java

```diff
@@ -84,19 +83,20 @@ public void doBefore(MethodInfo methodInfo, Context context) {
         RedirectProcessor.setTagsIfRedirected(Redirect.DATABASE, span, url);
         DatabaseInfo databaseInfo = DatabaseInfo.getFromConnection(conn);
         if (databaseInfo != null) {
-            span.remoteServiceName(remoteServiceName(databaseInfo.getDatabase()));
+            span.remoteServiceName(remoteServiceName(databaseInfo));
             span.remoteIpAndPort(databaseInfo.getHost(), databaseInfo.getPort());
         }
         span.start();
         context.put(SPAN_KEY, span);
     }
 
-    public String remoteServiceName(String database) {
-        if (StringUtils.isEmpty(database)) {
-            return "mysql";
-        } else {
-            return "mysql-" + database;
-        }
+    public String remoteServiceName(DatabaseInfo info) {
+        return String.format("%s:%s", info.getDatabaseType(), info.getDatabase());
```

Comment:
```suggestion
        return String.format("%s-%s", info.getDatabaseType(), info.getDatabase());
```
If not necessary, please maintain the formatting.